### PR TITLE
Add PlayerUiState

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Add `PlayerUiState`.

#### WHY

In order to have a class to represent the state of the Media UI components.

#### HOW
- Add `PlayerUiState`;
- Add `MediaItemUiModel`;
- Add `TrackPositionUiModel`;

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

**Comment by luizgrp** [comment]
cc: @dpreussler

File: media-ui/src/main/java/com/google/android/horologist/media/ui/state/model/MediaItemUiModel.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.horologist.media.ui.state.model
+
+public data class MediaItemUiModel(
+    val title: String? = null,
+    val artist: String? = null
```
Comment (yschimke) [inline media-ui/src/main/java/com/google/android/horologist/media/ui/state/model/MediaItemUiModel.kt:21]: No problem for now.  But we should consider how this expands.

Do we eventually want fields like album? Is it safe as a data class?

File: media-ui/src/main/java/com/google/android/horologist/media/ui/state/model/TrackPositionUiModel.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.horologist.media.ui.state.model
+
+public data class TrackPositionUiModel(
+    val current: Long,
```
Comment (yschimke) [inline media-ui/src/main/java/com/google/android/horologist/media/ui/state/model/TrackPositionUiModel.kt:20]: I do wonder whether Long is right type here, although not sure Duration works well for current.

**Review by yschimke — APPROVED** [review]

File: media-ui/src/main/java/com/google/android/horologist/media/ui/state/model/TrackPositionUiModel.kt (hunk 2 above)
Comment (luizgrp) [reply media-ui/src/main/java/com/google/android/horologist/media/ui/state/model/TrackPositionUiModel.kt:20]: tracked in b/228414169

**Pull request merged**